### PR TITLE
Make CRSF and SBUS as one module with 2 protocols

### PR DESCRIPTION
**Pull request by howard0su**
CRSF and SBUS are largely same. So merge the code to use one source file.

### CODE REVIEW

**Comment by howard0su** [comment]
Need PR #474 to free up some space in order to pass the build.

File: src/protocol/crsf_uart.c [hunk 1]
```diff
@@ -317,13 +370,25 @@ static void initialize()
     Transmitter.audio_player = AUDIO_DISABLED; // disable voice commands on serial port
 #endif
     UART_Initialize();
-    UART_SetDataRate(CRSF_DATARATE);
-    UART_SetDuplex(UART_DUPLEX_HALF);
+
+    if (Model.protocol == PROTOCOL_CRSF)
+    {
+        UART_SetDataRate(CRSF_DATARATE);
+        UART_SetFormat(8, UART_PARITY_NONE, UART_STOPBITS_1);
```
[the remark this answers is not in the document]
Comment (hexfet) [reply src/protocol/crsf_uart.c]: No need.  Those are the default values.

**Comment by howard0su** [comment]
@hexfet  why the following line calls Initialize instead of UART_Stop()?
 case PROTOCMD_DEINIT: UART_Initialize(); return 0;

**Comment by hexfet** [comment]
It probably could be UART_Stop now since audio is now initialized on model load.  Some cleanup of uart usage is on my todo list.  See [this post](https://www.deviationtx.com/forum/protocol-development/7474-serial-out-for-lrs-boxes?start=80#69082).

**Comment by howard0su** [comment]
I will leave that to you and not touch it in this PR.

**Pull request closed without merging**